Makefile

```diff
@@ -11,3 +11,7 @@ gofumpt:
 fixalign:
 	go install golang.org/x/tools/go/analysis/passes/fieldalignment/cmd/fieldalignment@latest
 	fieldalignment -fix $(filter-out $@,$(MAKECMDGOALS)) # the full package name (not path!)
+
+.PHONY: protoc
+protoc:
+	protoc --go_out=.  messages/proto/messages.proto
```

Comment:
You can simplify this with:
```go
protoc --go_out=./ --go-grpc_out=./ messages/proto/*.proto
```

Reply:
Simplified

[7cc28e2](https://github.com/madz-lab/go-ibft/pull/10/commits/7cc28e2c27e628d64c036776874c16fbff490de0)